Project-Timer-frontend/src/app/login-page/login-page.component.ts

```diff
@@ -0,0 +1,36 @@
+import {Component, OnInit} from '@angular/core';
+import {AbstractControl, FormControl, Validators} from '@angular/forms';
+
+/**
+ * @title Input with error messages
+ */
```

Comment:
?

Reply:
To było w kodzie który skopiowałem, wiem że to jest komentarz ale myślałem że on jest ważny z jakiegoś powodu - usunę.